tools/build_utils.py

```diff
@@ -680,7 +680,7 @@ def build_openvino_tf(build_dir, artifacts_location, ovtf_src_loc, venv_dir,
         cmake_cmd.extend(cmake_flags)
         cmake_cmd.extend([ovtf_src_loc.replace("\\", "\\\\")])
     else:
-        cmake_cmd = ["cmake"]
+        cmake_cmd = ["cmake", "-DCMAKE_BUILD_TYPE=Release"]
```

Comment:
Do not make this Release by default. Sometimes we may need to build in debug mode. Pass `arguments.debug_build` from build_ovtf.py to this function to decide here if it should be Release or Debug. https://github.com/openvinotoolkit/openvino_tensorflow/blob/4c6bfa75657d7d95fc018f9d3a3878740d112ecf/build_ovtf.py#L491

Reply:
Sure, added a check for debug build enabled or not in build_utils.py. If it is disabled then only it adds CMAKE_BUILD_TYPE=Release, because for Debug mode it is already handled in build_ovtf.py https://github.com/openvinotoolkit/openvino_tensorflow/blob/4c6bfa75657d7d95fc018f9d3a3878740d112ecf/build_ovtf.py#L492